src/zipWithIndex.js

```diff
@@ -0,0 +1,3 @@
+const zipWithIndex = (xs) => xs.map((x, i) => [x, i]);
```

Comment:
handle situation with  xs === null

write a test for it in `tests/zipWithIndex.js`

Reply:
Added check and test
@AllDayAlone